packages/compiler/test/decorators/decorators.test.ts

```diff
@@ -569,25 +569,26 @@ describe("compiler: built-in decorators", () => {
       ok(!getOverloadedOperation(runner.program, compiled.someUnrelatedThing));
       const overloadedBy = getOverloads(runner.program, compiled.someThing)?.map((op) => op.name);
       ok(overloadedBy?.length == 2);
-      ok(overloadedBy?.indexOf("someStringThing") !== -1);
-      ok(overloadedBy?.indexOf("someNumberThing") !== -1);
+      ok(overloadedBy?.includes("someStringThing"));
+      ok(overloadedBy?.includes("someNumberThing"));
       ok(getOverloads(runner.program, compiled.someUnrelatedThing) === undefined);
     });
 
     it("can overload operations defined in a namespace", async () => {
       const compiled = (await runner.compile(`
-        namespace ADifferentNS {
+        namespace MyArea {
           @test
           op someThing(param: string | int32): string | int32;
 
           @test
           @overload(someThing)
           op someStringThing(param: string): string;
+
+          @test
+          @overload(MyArea.someThing)
+          op someNumberThing(param: int32): int32;
         }
 
-        @test
-        @overload(ADifferentNS.someThing)
-        op someNumberThing(param: int32): int32;
```

Comment:
I updated this spec to explicitly test the opposite, it doesn't make much sense to me that overloads can be defined in different places. You should have the entire picture of all the overloads you want.